src/config.erl

```diff
@@ -183,6 +187,26 @@ delete(Section, Key, Persist, Reason) when is_list(Section), is_list(Key) ->
     gen_server:call(?MODULE, {delete, Section, Key, Persist, Reason}).
 
 
+features() ->
+    lists:usort([list_to_binary(Key) || {Key, "true"} <- ?MODULE:get(?FEATURES)]).
+
+
+feature_enable(Feature) when is_atom(Feature) ->
```

Comment:
or maybe `add_feature` / `remove_feature`. I think you're only supporting `remove_feature` for testing?